### PR TITLE
Responding to issues 38

### LINKED ISSUE #38: Correction due to specification change of Personium Core Request message

### PR DESCRIPTION
**Pull request by csKyou**
Close #38

### CODE REVIEW

File: src/html/js/common_personium.js [hunk 1]
```diff
@@ -308,11 +308,13 @@ Common.sendMessageAPI = function(uuid, extCell, type, title, body, reqRel, reqRe
     data.Title = title;
     data.Body = body;
     data.Priority = 3;
-    if (reqRel) {
-        data.RequestRelation = reqRel;
-    }
-    if (reqRelTar) {
-        data.RequestRelationTarget = reqRelTar;
+    if (reqType) {
```
Comment (dixonsiu) [inline src/html/js/common_personium.js:316]: What happen when reqType is null?

File: src/html/js/common_personium.js [hunk 2]
```diff
@@ -298,7 +298,7 @@ Common.appendRequestCells = function(extUrl, dispName) {
     $("#bSendAllowed").prop("disabled", false);
 };
 
-Common.sendMessageAPI = function(uuid, extCell, type, title, body, reqRel, reqRelTar) {
+Common.sendMessageAPI = function(uuid, extCell, type, title, body, reqType, reqRel, reqRelTar) {
```
Comment (dixonsiu) [inline src/html/js/common_personium.js]: Please add the following comment to explain why suddenly we don't need to specify App URL anymore.  

    /*
     * When the following conditions are satisfied, there is no need to include App URL when specifying the role/relation name.
     * 1. BoxBound must set to true
     * 2. Authorization token must be App authenticated token
     */

File: src/html/js/common_personium.js (hunk 1 above)
Comment (csKyou) [reply src/html/js/common_personium.js:316]: When reqType is null, we ignore RequestObjects because type is "message".

Comment (dixonsiu) [reply src/html/js/common_personium.js:316]: I understood. The following 2 functions actually does not require the last 3 arguments.  
1. Common.approvalRel  
1. Common.rejectionRel

**Review by dixonsiu — APPROVED** [review]

**Pull request merged**